includes/Products/Stock.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package FacebookCommerce
+ */
+
+namespace SkyVerge\WooCommerce\Facebook\Products;
+
+defined( 'ABSPATH' ) or exit;
+
+use SkyVerge\WooCommerce\Facebook\Products;
+
+/**
+ * The product stock handler.
+ *
+ * @since 2.0.4-dev.1
+ */
+class Stock {
+
+
+	/**
+	 * Stock constructor.
+	 *
+	 * @since 2.0.4-dev.1
+	 */
+	public function __construct() {
+
+		$this->add_hooks();
+	}
+
+
+	/**
+	 * Adds needed hooks to keep product stock in sync.
+	 *
+	 * @since 2.0.4-dev.1
+	 */
+	public function add_hooks() {
```

Comment:
```suggestion
	private function add_hooks() {
```

is there a reason for this to be public?

Reply:
Not really. It was copy pasta.